### DISCUSSION TITLE
Release v0.3.0

### DISCUSSION THREAD

**Discussion by alvin-reyes** [application-research/estuary#755] [Announcements]
## What's Changed
* reverted code to use direct FindInBatches call by @jlogelin in https://github.com/application-research/estuary/pull/745
* use debug level for logging this message by @snissn in https://github.com/application-research/estuary/pull/748
* refactor to use nsq and websocket by @en0ma in https://github.com/application-research/estuary/pull/744
* dont silent pin dequeue errors by @en0ma in https://github.com/application-research/estuary/pull/749
* Refactor pin estuary native: part 1 by @anjor in https://github.com/application-research/estuary/pull/746
* make sanity check more deterministic by @en0ma in https://github.com/application-research/estuary/pull/750
* Bump github.com/ipfs/go-bitswap from 0.10.2 to 0.11.0 by @dependabot in https://github.com/application-research/estuary/pull/751
* Prevent duplicating last batch in each batched append pattern by @neelvirdy in https://github.com/application-research/estuary/pull/754

**Full Changelog**: https://github.com/application-research/estuary/compare/v0.2.5...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/application-research/estuary/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by alvin-reyes** [comment]
This is the first release that we will provide an option for estuary app owners to use a QUEUE instead of an RPC communication protocol between the API and Shuttle nodes. For more information about the difference, you can read up the following article here: https://particular.net/blog/rpc-vs-messaging-which-is-faster#threads-and-memory

Add the following flags on the API and Shuttle node 
```
--queue-eng-enabled --queue-eng-host=ip:4150 --queue-eng-consumers=1
```

Current estuary configuration:
```
--queue-eng-enabled --queue-eng-host=<queue IP>:4150 --queue-eng-consumers=3
```

**Comment by MatiasDiazDamian** [comment]
Hi. Sorry for the mistake but I haven't texted you.
